docs/install_guides/installing-openvino-from-archive-linux.md

```diff
@@ -16,9 +16,7 @@ See the [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNo
   * Ubuntu 18.04 long-term support (LTS) x86, 64-bit
   * Ubuntu 20.04 long-term support (LTS) x86, 64-bit
   * Red Hat Enterprise Linux 8 x86, 64-bit
-
-  .. note::
-     Since the OpenVINO™ 2022.1 release, CentOS 7.6, 64-bit is no longer supported.
+  * CentOS 7 x86, 64-bit (Note that `devtoolset-7` or a higher version is required to compile applications with OpenVINO)
 
 .. tab:: Hardware
 
```

Comment:
I believe that also "Software" tab should be updated with Python 3.10 version

Reply:
> I believe that also "Software" tab should be updated with Python 3.10 version

yeah, it's been fixed in another PR https://github.com/openvinotoolkit/openvino/pull/14672 . Updated. :)